students/mint_k/lesson06/good_perf.py

```diff
@@ -0,0 +1,52 @@
+"""
+Lesson06
+
+"""
+
+import datetime
+import csv
+
+
+def analyze(filename):
+    """this is to hash through the data"""
+    start = datetime.datetime.now()
+    with open(filename) as csvfile:
+        reader = csv.reader(csvfile, delimiter=',', quotechar='"')
+        found = 0
+        year_count = {
```

Comment:
You could simplify it here by making it figure out which years are present in the data:

```
        year_count = {}
        for row in reader:
            try:
                year_count[row[5][-4:]] += 1
            except KeyError:
                year_count[row[5][-4:]] = 1
                continue
```